jarviscli/plugins/nationalize.py

```diff
@@ -0,0 +1,19 @@
+import requests
+from colorama import Fore
+from plugin import plugin, require
+
+@require(network=True)
+@plugin("nationalize")
+def nationalize(jarvis, s):
+    """Tells the nationality of someone based on his name"""
```

Comment:
Even though it appears not to be required, it is always good idea to add a "powered by" in docstring

Reply:
Sorry, was unclear; actually meant a 'powered by www.boredapi.com' ;). Small note: Always good idea to check PEP8:

```
flake8 --select E,W --max-line-length=140 --ignore E722,W503,W504,E128 jarviscli/ installer
jarviscli/plugins/agify.py:5:1: E302 expected 2 blank lines, found 1
jarviscli/plugins/agify.py:19:1: W293 blank line contains whitespace
```

But don't mind ;), I'll just push a small commit.